lib/screens/admin_new_product_page.dart

```diff
@@ -493,6 +494,72 @@ class _AdminNewProductPageState extends State<AdminNewProductPage> {
     );
   }
 
+  Widget _buildDeleteStockItemButton(Stock stock) {
+    return Row(
```

Comment:
I think you need to add padding to make the delete icon look symmetrical with respect to the stock tile. Please refer to https://github.com/GospelAid/badiup/pull/120#issuecomment-588223870